db/schema.rb

```diff
@@ -49,6 +49,7 @@
     t.datetime "updated_at"
     t.datetime "applied_at"
     t.integer  "updater_id"
+    t.text     "state",                       default: "draft", null: false
```

Comment:
I think the new default column is now `aasm_state` (which also should also remove the need for the  `column: :state` option in the model). Can you change it for the less generic name? It could also be a string column instead of a text one, but I guess that makes little difference